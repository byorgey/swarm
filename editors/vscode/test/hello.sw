// SYNTAX TEST "swarm" "sample testcase"

def hello: Text = "Hello World" end;

return hello;
