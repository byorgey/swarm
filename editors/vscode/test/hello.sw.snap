>// SYNTAX TEST "swarm" "sample testcase"
#^^ source.swarm comment.line.double-slash
#  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.swarm comment.line.double-slash
>
>def h: text = "Hello World" end;
#^^^^ source.swarm keyword.control.dictionary.def
#    ^ source.swarm keyword.control.dictionary.def entity.name.function
#     ^ source.swarm keyword.control.dictionary.def
#      ^^^^^^ source.swarm keyword.control.dictionary.def entity.name.type
#            ^ source.swarm keyword.control.dictionary.def
#             ^ source.swarm keyword.control.dictionary.def
#              ^ source.swarm keyword.control.dictionary.def string.quoted.double
#               ^^^^^^^^^^^ source.swarm keyword.control.dictionary.def string.quoted.double
#                          ^ source.swarm keyword.control.dictionary.def string.quoted.double
#                           ^ source.swarm keyword.control.dictionary.def
#                            ^^^ source.swarm keyword.control.dictionary.def
#                               ^^ source.swarm
>
>return "";
#^^^^^^ source.swarm keyword.other
#      ^ source.swarm
#       ^ source.swarm string.quoted.double
#        ^ source.swarm string.quoted.double
#         ^^ source.swarm
>